projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,128 @@
+
+  
+You can read more about our journey on [our blog](https://www.entidad.io/blog). 
+
+# Alignment with the OpenWallet Foundation Mission
+The **Farmworker Wallet OS** project is a community of contribution led by Entidad and the United Farm Workers Foundation (UFWF) with the goal of furthering the adoption of an open, secure, interoperable [digital wallet solution](https://youtu.be/Uo7CTtGrOw8) that makes it easier for farm worker communities to access life-changing social and human services. The open-source software components maintained by this community will enable non-profit and social good organizations to build wallet-driven solutions on top of the Mendix low-code platform. Collaboration with app makers from the broader Mendix developer community will help inform and guide the Farmworker Wallet OS project with industry proven best practices, leading to the implementation of reusable functionality that is aligned with the goals of the Open Wallet Foundation (to preserve user choice, security and privacy). The community will also inspire the next generation of low-code app builders to learn and expand their skill sets with decentralized (web3) technologies. 
+
+# Current Code of Conduct
+[Farmworker Wallet OS Community Code of Conduct](https://github.com/Entidad/.github/blob/main/CODE_OF_CONDUCT.md)
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+# Project License
+[Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+
+# Source Control
+Github
+
+# Issue Tracker
+Github
+
+# External Dependencies
+Mendix is a low-code application platform provider so its terms of service cover usage of Mendix services and resources by Mendix developers (organizations and/or individual contributors). The **Application model**  for a standards compliant wallet engine will be the primary focus for the contributors of this project. The Mendix terms of service do protect the IP rights to these App models in Section 3 with reference to "Customer Data" definition in section 17.
+
+There are [design-time](https://www.mendix.com/evaluation-guide/developing-in-mendix/) and [runtime](https://www.mendix.com/evaluation-guide/app-lifecycle/deployment/) aspects of app development but one of the great things about this tooling is that there is clean separation between the two. 
+
+The following captures the developer's "design-time" perspective. We hope that the diagram helps to clarify the software components that would fall under the scope of this OWF project and distinguishes Preparese<sup>TM</sup> as an example of a closed (proprietary) application that embeds core Farmworker WalletOS components. We plan on documenting a similar diagram to aid in understanding the "runtime" perspective soon.
```

Comment:
Recommend to use more specific repository names (example `demo-native-template` and `demo-dev-native-app` seem too generic to me and could conflict with other projects / confuse folks).

Reply:
Repository names for reference implementation components have been renamed.